09-mapping.Rmd

```diff
@@ -182,8 +197,10 @@ tm_shape(nz) + tm_fill(col = 1:nrow(nz)) # fails:
 ```
 
 Instead `col` (and other aesthetics that can vary such as `lwd` for line layers and `size` for point layers) requires a character string naming an attribute associated with the geometry to be plotted.
-Thus one would achieve the desired (plotted in the right-hand panel of Figure \@ref(fig:tmcol)) result as follows:^[
-Figure \@ref(fig:tmcol) also demonstrates another benefit of **tmap**: it automatically converts numeric variables into sensible, graduated bins.
+Thus, one would achieve the desired (plotted in the right-hand panel of Figure \@ref(fig:tmcol)) result as follows:^[
+<!-- wouldn't say sensible, since here it appears as if tmap simply uses equal class sizes...-->
+Figure \@ref(fig:tmcol) also demonstrates another benefit of **tmap**:
```

Comment:
wouldn't say sensible, since here it appears as if tmap simply uses equal class sizes...